pytta/_plot.py

```diff
@@ -82,7 +82,7 @@ def time(sigObjs, xLabel, yLabel, yLim, xLim, title, decimalSep, timeUnit):
         except:
             locale.setlocale(locale.LC_NUMERIC, 'C')
             plt.rcParams['axes.formatter.use_locale'] = False
-            print("Plotting error using ',', then was used '.' as decimal separator")
+            ValueError("Plotting error using ',', then was used '.' as decimal separator")
```

Comment:
The correct syntax would be: raise ValueError(" ...
But in this case the rest of the code wouldn't executed. I think the print version is better. With the improvement I mentioned in the other comment I think It will be enough.

Reply:
If an error is raised, there is no reason to wrap it in a try-except block. The print was ok as some sort of warning.